main.py

```diff
@@ -1 +1,7 @@
+def say_hello(person_name):
+    print(f"Hi, It is so nice to be collaborating with you {person_name}.")
```

Comment:
Punctuation nitpick -- maybe we want a comma here?

```suggestion
    print(f"Hi, It is so nice to be collaborating with you, {person_name}.")
```

Reply:
Yes. That comma does in fact belong there! 😅